Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/Core/Theme.cs

```diff
@@ -15,7 +15,6 @@ namespace Microsoft.MixedReality.Toolkit.UI
     public class Theme : ScriptableObject
     {
         [FormerlySerializedAs("Settings")]
-        [FormerlySerializedAs("Definitions")]
```

Comment:
Why removed?

Reply:
Definitions was only used in this feature branch between PRs. Outside of this feature branch, it has always been "Settings"